app/src/features/observations/components/BlockObservationForm.tsx

```diff
@@ -0,0 +1,750 @@
+import Box from '@material-ui/core/Box';
+import Grid from '@material-ui/core/Grid';
+import TextField from '@material-ui/core/TextField';
+import { useFormikContext } from 'formik';
+import React, { useState } from 'react';
+import makeStyles from '@material-ui/core/styles/makeStyles';
+import FormControl from '@material-ui/core/FormControl';
+import InputLabel from '@material-ui/core/InputLabel';
+import Select from '@material-ui/core/Select';
+import MenuItem from '@material-ui/core/MenuItem';
+import FormHelperText from '@material-ui/core/FormHelperText';
+import { DATE_LIMIT } from 'constants/dateFormats';
+import { HotTable } from '@handsontable/react';
+import Handsontable from 'handsontable';
```

Comment:
You have a failing test here .. not sure what's causing it to fail on the local instance, but not in Openshift 
![Screen Shot 2021-06-14 at 10 54 27 PM](https://user-images.githubusercontent.com/22803003/122000494-c4137d00-cd63-11eb-9a4a-33288ea0d817.png)

Reply:
the tests arent failing for me and also arent failing on the CI checks...

Reply:
`Cannot find module` errors are likely a result of needing to run `npm install`